edb/server/protocol/execute.pyx

```diff
@@ -611,6 +611,8 @@ class DecimalEncoder(json.JSONEncoder):
                 ) + '}'
         if isinstance(obj, list):
             return '[' + ', '.join(map(self.encode, obj)) + ']'
+        if isinstance(obj, bytes):
+            return base64.b64encode(obj).decode()
```

Comment:
I think the string needs to be wrapped in `"` so it's valid json?

Reply:
Yep, just recursed on `self.encode` again, but maybe it's better to just wrap it in quotes? wdyt?

Reply:
I guess wrapping in quotes is slightly more efficient, since we already know it's a string type with no characters that need escaping, but it's probably only a marginal difference.